docs/software/admin.md

```diff
@@ -515,14 +515,18 @@ Until the node sees a quorum, it will say
 After observing consensus, a new field `quorum` will be set with information on what the network decided on, at this point the node will switch to "*Catching up*":
 ```json
       "quorum" : {
-         "22267866" : {
+         "qset" : {
+            "ledger" : "22267866",
             "agree" : 5,
             "delayed" : 0,
             "disagree" : 0,
             "fail_at" : 3,
             "hash" : "980a24",
             "missing" : 0,
             "phase" : "EXTERNALIZE"
+         },
+         "transitive" : {
```

Comment:
missing documentation for `intersection`, should tell people to check logs and take immediate action if value is 0.
I wonder if we should also include a `count` (or `size`) field here (size of the transitive quorum)

Reply:
Fixed.